mesonbuild/wrap/wraptool.py

```diff
@@ -71,8 +71,11 @@ def search(options: 'argparse.Namespace') -> None:
     name = options.name
     releases = get_releases()
     for p in releases.keys():
-        if p.startswith(name):
+        if p.find(name) != -1:
             print(p)
+        for dep in releases.get(p).get('dependency_names'):
```

Comment:
Better do `for p, info in releases.items()` and then `for dep in info.get('dependency_names', [])`.

Reply:
Maybe should not look in dependencies if the name matched already, which probably would happen most of the time.